src/test-apps/happy/lib/WeaveDeviceManager.py

```diff
@@ -147,9 +162,38 @@ class ExtendedOption (Option):
 
     try:
         if options.useBle:
+            from WeaveBluezMgr import BluezManager as BleManager
+            from WeaveBleUtility import FAKE_CONN_OBJ_VALUE
+            bleManager = BleManager(devMgr)
+
+            try:
+                bleManager.ble_adapter_select(identifier=options.bleSrcAddr)
+            except WeaveDeviceMgr.DeviceManagerException, ex:
+                print ex
+                exit()
+
+            try:
+                line = ' -t 120 ' + str(options.bleDstAddr)
```

Comment:
Lets not do that.  I would really prefer we did not append random timeouts to the command line

Reply:
Done, thanks